custom/icds_reports/tasks.py

```diff
@@ -987,8 +987,15 @@ def _get_value(data, field):
     return getattr(data, field) or default
 
 
-@periodic_task(run_every=crontab(minute=30, hour=18), acks_late=True, queue='icds_aggregation_queue')
-def collect_inactive_awws():
+# This task causes issues on the india environment
+# so it's limited to only ICDS_ENVS
+if settings.SERVER_ENVIRONMENT in settings.ICDS_ENVS:
+    @periodic_task(run_every=crontab(minute=30, hour=18), acks_late=True, queue='icds_aggregation_queue')
+    def collect_inactive_awws():
+        _collect_inactive_awws()
```

Comment:
If we think that it would be fine/better to block _all_ periodic tasks in this file form running on all envs but `ICDS_ENVS`, then we could instead of this put at the beginning of the file

```
if settings.SERVER_ENVIRONMENT in settings.ICDS_ENVS:
    icds_periodic_task = periodic_task
else:
    icds_periodic_task = lambda **kwargs: lambda fn: fn
```

and use `@icds_periodic_task` throughout. This may even be a better pattern for just this one spot, actually. I'm going to change it—but still let me know if you think I should/can apply it to the rest of the file as well.